gobblin-utility/src/main/java/gobblin/util/StringUtils.java

```diff
@@ -0,0 +1,66 @@
+package gobblin.util;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+public class StringUtils {
+
+  public static final Pattern HUMAN_READABLE_SIZE_PATTERN = Pattern.compile("([0-9\\.]+)\\s*([kKmMgGtTpP]?)[bB]?");
+
+  /**
+   * Convert a human readable string (e.g. 10kb) into the number of bytes.
+   *
+   * Examples: 10b, 10kb, 10mb, 10gb, 10tb, 10pb, 1.2m, ...
+   */
+  public static long humanReadableToByteCount(String string) throws FormatException {
```

Comment:
Is this used? Also, can you change the class name? `StringUtils` is too general.